SIT.Manager/Services/AppUpdaterService.cs

```diff
@@ -14,11 +14,11 @@
 using System.Text.Json;
 using System.Threading.Tasks;
 
-namespace SIT.Manager.Avalonia.Services;
+namespace SIT.Manager.Services;
 
 public class AppUpdaterService(IFileService fileService, ILogger<AppUpdaterService> logger, HttpClient httpClient, IManagerConfigService managerConfigService) : IAppUpdaterService
 {
-    private const string MANAGER_VERSION_URL = @"https://api.github.com/repos/stayintarkov/SIT.Manager.Avalonia/releases/latest";
+    private const string MANAGER_VERSION_URL = @"https://api.github.com/repos/stayintarkov/SIT.Manager/releases/latest";
```

Comment:
Is this URL still going to be correct?

Reply:
Nope... That's what I can for running a find and replace without checking,,,